### PR TITLE
fix wrong tag in json_flag.json

### PR DESCRIPTION
**Pull request by cainiaowu**
Also, description field should really rename to comment, and info to description, otherwise it got extracted by pot script.

### CODE REVIEW

File: data/json/flags.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "id": "FLOTATION",
     "type": "json_flag",
     "context": [ "ARMOR", "TOOL_ARMOR" ],
-    "info": "This clothing <neutral>prevents going underwater</info> including both <good>drowning</good> and <bad>diving</bad>."
+    "info": "This clothing <info>prevents going underwater</info> including both <good>drowning</good> and <bad>diving</bad>."
```
Comment (mugling) [inline data/json/flags.json]: Should be `neutral`

Comment (Zireael07) [inline data/json/flags.json]: If it should be neutral, then the closing tag must be </neutral>.

Comment (cainiaowu) [inline data/json/flags.json]: do we have any policy on <neutral>/<info> tag use?

Comment (mugling) [inline data/json/flags.json]: `neutral` is more consistent with other code

**Pull request merged**